lib/child_process.js

```diff
@@ -109,7 +97,6 @@ function normalizeExecArgs(command /*, options, callback*/) {
 exports.exec = function(command /*, options, callback*/) {
   var opts = normalizeExecArgs.apply(null, arguments);
   return exports.execFile(opts.file,
-                          opts.args,
```

Comment:
I'm not sure I understand your question. The `args` to `execFile()` is optional. Before this change, `args` would contain the arguments to the shell and the command. With this change, that happens in `normalizeSpawnArguments()`.